src/test/java/edu/cornell/kfs/concur/services/ConcurAccountValidationServiceTest.java

```diff
@@ -0,0 +1,444 @@
+package edu.cornell.kfs.concur.services;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kuali.kfs.sys.KFSConstants;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ValidationResult;
+import edu.cornell.kfs.concur.service.impl.ConcurAccountValidationServiceImpl;
+import static org.junit.Assert.assertEquals;
```

Comment:
Per Kuali standards, we would like to avoid static imports. We can just import the Assert class instead and change the method calls to Assert.assertEquals(...).